docs/cookbook/dynamic-configuration/01_Basics/01_Structure.md

```diff
@@ -0,0 +1,65 @@
+# 01 Structure
+
+Dynamic Configuration uses a series of files to load templates. This example will show how they all combine together. Running the below command will combine all the templates into the final.yml.
+
+`docker run -v ${PWD}/:/etc/grafana grafana/agentctl:latest template-parse /etc/grafana/02_config.yml`
+
+## Dynamic Configuration
+
+[config.yml](./01_config.yml)
+
+```yaml
+template_paths:
+  - "file:///etc/grafana/01_assets"
```

Comment:
is the file:// scheme required here?

Reply:
I can change this here in the template_paths and datasources to default to `file://` if none specified BUT datasources embedded in templates would still need need `file://` and that seems inconsistent.